app/helpers/gallery_helper.rb

```diff
@@ -14,6 +14,19 @@ def gallery_content(gallery)
     }
   end
 
+  def gallery_promo_content(gallery)
+    return nil if gallery.blank?
+
+    {
+      url: gallery_path(gallery),
+      title: gallery.title,
+      description: gallery.description,
+      images: gallery_promo_image_content(gallery),
+      type: t('global.promotions.gallery'),
+      count_label: t('site.object.promotions.card-labels.gallery-count', count: gallery.images.size)
```

Comment:
This should say "19 images" instead of "19 Items", just as it does on the galleries index page.